backend/elasticsearch/src/test/java/org/hibernate/search/backend/elasticsearch/gson/spi/GsonClassesTest.java

```diff
@@ -80,6 +80,9 @@ void testNoMissingGsonContractImplementations() {
 			for ( ClassInfo implementor : backendElasticsearchIndex.getAllKnownSubclasses( gsonContract ) ) {
 				classes.add( implementor.name() );
 			}
+			for ( ClassInfo implementor : backendElasticsearchIndex.getAllKnownImplementors( gsonContract ) ) {
```

Comment:
Since `TypeAdapterFactory` is an interface, none of those factories were actually added to the `classes` set 🙈